crates/ruff_linter/src/rules/pycodestyle/snapshots/ruff_linter__rules__pycodestyle__tests__E712_E712.py.snap

```diff
@@ -1,15 +1,15 @@
 ---
 source: crates/ruff_linter/src/rules/pycodestyle/mod.rs
 ---
-E712.py:2:11: E712 [*] Comparison to `True` should be `cond is True` or `if cond:`
+E712.py:2:11: E712 [*] Avoid equality comparisons to `True`; use `if cond:` for truth checks
   |
 1 | #: E712
 2 | if res == True:
   |           ^^^^ E712
 3 |     pass
 4 | #: E712
   |
-  = help: Replace with `cond is True`
+  = help: Replace with `cond`
```

Comment:
It's possibly outside the scope of this PR (you can leave it for now, if you like), but this suggestion is pretty confusing. There's no variable named "cond" in scope here, and I don't think it's obvious that this means "replace `if res == True` with `if res`", especially when the carets only point to `True` rather than `res == True`.